### PR TITLE
taavi uifixes

### LINKED ISSUE #631: Align of UI elements.
@pe0ny: is the align correct in this snapshots (not in my opinion)? In case of not, let me know what is the correct solution.

nexus 4 kitkat screentshot 1

nexus 4 kitkat screentshot 3

nexus 4 kitkat screentshot 2

### PR DESCRIPTION
**Pull request by taavirehemagi**
Closes #631, #661 #636

This is how the project selection looks like, now.
screen shot 2015-05-14 at 11 37 57

### CODE REVIEW

File: Joey/Resources/layout/LogTimeEntryListItem.axml [hunk 1]
```diff
@@ -12,7 +12,7 @@
         android:layout_width="4dp"
         android:layout_height="72dp"
         android:background="@drawable/ColoredSideShape"
-        android:layout_marginRight="12dp" />
+        android:layout_marginRight="28dp" />
```
Comment (monday8am) [inline Joey/Resources/layout/LogTimeEntryListItem.axml]: final result is like this? 
https://github.com/toggl/mobile/issues/652

Comment (taavirehemagi) [inline Joey/Resources/layout/LogTimeEntryListItem.axml]: No, it's like the design guideline. But I have the layout you linked in another PR. Which one should we choose?
74a0b796-ee54-11e4-829d-ada24b60e290

Comment (taavirehemagi) [inline Joey/Resources/layout/LogTimeEntryListItem.axml]: Currently, it's really cramped and doesn't fit a lot of info, but it looks great when it fits

Comment (taavirehemagi) [inline Joey/Resources/layout/LogTimeEntryListItem.axml]: screen shot 2015-05-14 at 12 11 39

**Comment by monday8am** [comment]
@taavirehemagi:
- Back to 16 dp margin in TE list screen. We need to show the content and this was the solution we found.
- "No project" is not a header. Is an option like the other projects and should have the same font size.
- The font for headers is not correct. Should be equal to TE list screen headers.

@pe0ny 
- Please, let us know about margins and colours in Project list page, 16dp and 32dp or 16dp all?
- Client name color?

**Comment by taavirehemagi** [comment]
screen shot 2015-05-15 at 11 55 52

**Comment by pe0ny** [comment]
* Margins seem correct - workspace 16dp and project 32 dp
* And I actually like that client has the same (text) color as project in this case.

**Comment by monday8am** [comment]
:+1: thanks @pe0ny

**Comment by monday8am** [comment]
@taavirehemagi please, rebase this branch to merge it.

**Comment by taavirehemagi** [comment]
done

**Pull request merged**